src/dotnet/APIView/APIViewWeb/LeanModels/ReviewListModels.cs

```diff
@@ -97,18 +105,21 @@ public class BaseListitemModel
         public string PackageName { get; set; }
         public string Language { get; set; }
     }
-
+    
     public class ReviewListItemModel : BaseListitemModel
     {
         public HashSet<string> Subscribers { get; set; } = new HashSet<string>();
         public List<ReviewChangeHistoryModel> ChangeHistory { get; set; } = new List<ReviewChangeHistoryModel>();
         public List<ReviewAssignmentModel> AssignedReviewers { get; set; } = new List<ReviewAssignmentModel>();
         public bool IsClosed { get; set; }
         public bool IsApproved { get; set; }
+        public NamespaceReviewStatus NamespaceReviewStatus { get; set; } = NamespaceReviewStatus.NotStarted;
```

Comment:
This is effectively a duplicate of IsApproved. Can you add note next to IsApproved that that should be deprecated in the future. We do not want to have redundancy in the data but you can do that change later to move all checks on IsApproved to NamespaceReviewStatus in a different PR.
@AlitzelMendez and @tjprescott FYI..